stories/avatar-group/avatar-group.stories.js

```diff
@@ -396,7 +399,7 @@ GroupType.parameters = {
     docs: {
         iframeHeight: 250,
         description: {
-            story: 'To use Group type of Avatar Group use the `--group-type` modifier class <i>(`fd-avatar-group--group-type` class)<i>.'
+            story: 'For Group type add the `fd-avatar-group--group-type` modifier class to the `fd-avatar-group` base class.'
```

Comment:
shouldn't avatars in overflow menu be centered? 

<img width="391" alt="image" src="https://user-images.githubusercontent.com/20265336/178948566-27b08fd1-4d42-4413-a88d-b310101d2810.png">

Reply:
@platon-rov I was not able to reproduce this issue in any of the browsers I have. Could you please check again?